frame/nfts/src/lib.rs

```diff
@@ -565,19 +576,29 @@ pub mod pallet {
 		///
 		/// Weight: `O(1)`
 		#[pallet::weight(T::WeightInfo::create())]
-		pub fn create(origin: OriginFor<T>, admin: AccountIdLookupOf<T>) -> DispatchResult {
+		pub fn create(
+			origin: OriginFor<T>,
+			admin: AccountIdLookupOf<T>,
+			config: CollectionConfig,
+		) -> DispatchResult {
 			let collection =
 				NextCollectionId::<T, I>::get().unwrap_or(T::CollectionId::initial_value());
 
 			let owner = T::CreateOrigin::ensure_origin(origin, &collection)?;
 			let admin = T::Lookup::lookup(admin)?;
 
+			let mut config = config;
+			// DepositRequired could be disabled by calling the force_create() only
+			if config.has_disabled_setting(CollectionSetting::DepositRequired) {
+				config.enable_setting(CollectionSetting::DepositRequired);
+			}
```

Comment:
```suggestion
config.enable_setting(CollectionSetting::DepositRequired);
```
I think it would be better to error here though. Otherwise the creator will wonder why it is enabled.

Reply:
Nice suggestion!